lib/commands/release/index.ts

```diff
@@ -1,6 +1,6 @@
 /**
```

Comment:
The output framework currently doesn't display multi-line values very nicely (see below).  I think our options here are to either: 

1. copy current behaviour, and indent subsequent lines to same column (Looks nice, but may hinder copy/paste)
2. add newline before outputting multi-line values (better copy/paste support)

Current:
```
IS FINAL:        true
BUILD LOG:       [Info]     Starting build for xxx, user xx
                 [Info]     Dashboard link: https://dashboard.balena-cloud.com/apps/xxx/devices
                 [Info]     Building on arm01
```
New framework (needs changing):
```
 Is final        : true
 Build log       : [Info]     Starting build for xxx, user xx
[Info]     Dashboard link: https://dashboard.balena-cloud.com/apps/xxx/devices
[Info]     Building on arm01
```

Reply:
Ok, let's go with option 2 then.